src/Controls/src/Core/WebView.cs

```diff
@@ -184,6 +184,10 @@ protected override void OnPropertyChanged(string propertyName)
 				if (source != null)
 					SetInheritedBindingContext(source, BindingContext);
 			}
+#if WINDOWS
+			else if (propertyName == nameof(Window) && Window is not null)
+				Window.Destroying += OnWindowDestroying;
+#endif
```

Comment:
This feels like the wrong place... or thing... no idea why just yet, but maybe I am feeling that some other event should go down the line when a window hosting a control is closed... Is there some Unloaded or some such event that gets to controls when a window is closed.

If I have a label in a window and close the window, do we have some event? Unloaded, the Window property goes null.

Reply:
Mnn, yeah. I think that to have something simpler, we need a way to be able to have window change notifications from Handlers.